### ISSUE TITLE
Changing parameters in Alphalink2

### ISSUE THREAD

**Issue by soumyaremesh** [Rappsilber-Laboratory/AlphaLink2#28]
We are very excited to use Alphalink2 to predict antigen-antibody structures for several complexes where we have DSSO based cross linking data. Alphalink2 is a really useful software and fits well into our workflow. Thank you for this open source platform. 
I have a locally installed version of Alphalink2 and have successfully generated models for a complex of interest and also experimentally validated the predicted interface. I didn't see notes on how to change parameters like recycling iterations and sampling in the README file in the local version. These would be very useful for some of the other complicated complexes we are working on. Many thanks!

**Comment by lhatsk** [comment]
Hi!

Sorry, the parameters are not directly exposed at the moment. They can be changed in or via inference.py

https://github.com/Rappsilber-Laboratory/AlphaLink2/blob/main/inference.py#L329

https://github.com/Rappsilber-Laboratory/AlphaLink2/blob/main/inference.py#L334

or added to run_alphalink.sh e.g., 

```
python inference.py \
	--model_name="multimer_af2_crop" \
	--param_path=$param_path \
	--data_dir=$output_dir_base \
	--target_name=$target_name \
	--output_dir=$output_dir_base \
        --crosslinks=$crosslinks \
	--bf16 \
	--use_uniprot \
        --save_raw_output \
	--relax \
        --times=100 \
        --max_recycling_iters=20
```

I should expose these... For more complicated complexes it also makes sense to test the v3 weights. Hope AlphaLink2 helps you! Keep in mind it wasn't optimised for DSSO.

**Comment by lhatsk** [comment]
The options are now exposed in run_alphalink.sh

**Comment by soumyaremesh** [comment]
Just had a quick follow-up question about the method not being optimized for DSSO. Our assumption was that since the Ca-Ca distance limit for DSSO is comparable to that for SDA we could use DSSO. Are there other considerations to be made here?
Also, many thanks for such prompt responses and edits to the code.

**Comment by lhatsk** [comment]
No. It was just to manage expectations. SDA is already fairly low resolution at least compared to the evolutionary information used in structure prediction and DSSO data is usually sparser than SDA. If there is not much support in the evolutionary data, it's sometimes hard to get the structures close enough to form an interface.
